blocks/library/cover-image/index.js

```diff
@@ -183,6 +255,11 @@ registerBlockType( 'core/cover-image', {
 		return (
 			<section className={ classes } style={ style }>
 				<h2>{ title }</h2>
+				{ hasButton && <div className="button-container aligncenter" style={ { backgroundColor: buttonBackgroundColor } }>
```

Comment:
At a glance, it seems like we shouldn't need a container element here, that we could style the anchor alone and retain an ideal markup structure to save in the post.